packages/samples/headless-react/src/App.tsx

```diff
@@ -121,9 +124,11 @@ const history = buildHistory(engine);
 
 const relevanceInspector = buildRelevanceInspector(engine);
 
-const {autoUpdateURI: startUpdatingURI} = bindSearchParametersToURI(engine);
-
 function App() {
+  const {autoUpdateURI: startUpdatingURI} = bindSearchParametersToURI(engine, {
+    executeInitialSearch: true,
+  });
```

Comment:
Without executing an initial search, the category facets in the search parameters disappears.